napari/_qt/qt_resources/_svg.py

```diff
@@ -52,7 +51,6 @@ def __init__(
         colorized = get_colorized_svg(path_or_xml, color, opacity)
         super().__init__(SVGBufferIconEngine(colorized))
 
-    @lru_cache
```

Comment:
I have removed the cache as the returned object is mutable and could be changed.